packages/common/src/eips/4200.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "EIP-4200",
+  "number": 4200,
+  "comment": "Static relative jumps",
+  "url": "https://eips.ethereum.org/EIPS/eip-4200",
+  "status": "Review",
+  "minimumHardfork": "berlin",
+  "requiredEIPs": [3540],
```

Comment:
This should point to 3670 I think, since the EIP expands on the 3670 rule set. 3540 requirement is then implicitly added as well (along the 3670 activation).

Reply:
You are right, great catch, it is also in the EIP!